packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -885,7 +868,8 @@ class HotRunner extends ResidentRunner {
             nullSafety: usageNullSafety,
             fastReassemble: null,
           ).send();
-          return OperationResult(1, 'Reload rejected');
+          final ReloadReportContents contents = ReloadReportContents.fromReloadReport(reloadReport);
+          return OperationResult(1, 'Reload rejected: ${contents.notices.join("\n")}');
```

Comment:
including the additional context in the operation result is required to surface this information in IDEs